src/routes/adminPanel/adminPanelTable/table.component.tsx

```diff
@@ -0,0 +1,102 @@
+import { useState } from 'react';
```

Comment:
move this file and tablePagination to shared/components, as those components could be reusable

Reply:
also, I'm not sure if adminPanelTable should be in routes